enterprise/server/remote_execution/copy_on_write/copy_on_write.go

```diff
@@ -72,6 +74,11 @@ func updateMmappedBytesMetric(delta int64, fileNameLabel string) {
 		Set(float64(atomic.AddInt64(gaugeValPtr.(*int64), delta)))
 }
 
+type usageSummary struct {
+	totalCount        int
+	totalDurationUsec int64
```

Comment:
nit: store this as a `time.Duration` ? for duration values it's usually cleanest to store as `time.Duration` for as long as you can, then only convert it to a plain number when you have to call a func that doesn't accept a duration.